packages/app-security/src/admin/plugins/installation.tsx

```diff
@@ -156,6 +171,15 @@ const Install = ({ onInstalled }) => {
                                     )
                                 }
                             })}
+                            {!authUserMessage && (
+                                <Grid>
+                                    <Cell span={12}>
+                                        <Bind name="subscribed">
+                                            <Checkbox label={t`Subscribe to Newsletter`} />
```

Comment:
@SvenAlHamad @EmilK15 should we add some more text here so user knows what this is?

Reply:
I agree! @EmilK15 please set the message to something like this:
```
I want to receive updates on product improvements and new features. Doing so I accept Webiny's [privacy policy](https://www.webiny.com/privacy-policy/). 
```